packages/code-du-travail-frontend/src/outils/DureePreavisRetraite/stepReducer.tsx

```diff
@@ -25,7 +25,14 @@ export const initialState: State = {
       component: Steps.Informations,
       label: "Informations",
       name: "infos",
-      skip: (values) => !values.ccn || !supportedCcn.includes(values.ccn.num),
+      skip: (values) => {
+        if (
+          values.ccn.num === 44 &&
+          values["contrat salarié - mise à la retraite"] === "non"
+        )
+          return true;
```

Comment:
I'm not a huge fan of this hack. The idea is to see all specifics cases here then improve the skipped step.